taui/src/components/custom-sign-in.js

```diff
@@ -42,80 +24,23 @@ class SignInHeader extends React.Component {
-  // Based on source:
-  // https://github.com/aws-amplify/amplify-js/blob/master/packages/aws-amplify-react/src/Auth/SignIn.jsx#L120
-  showComponent (theme) {
-    const { authState, hide = [], federated, onStateChange, onAuthEvent, override = [], t } = this.props
-    const hideForgotPassword = !override.includes('ForgotPassword') &&
-      hide.some(component => component === ForgotPassword)
-
+class CustomSignIn extends React.Component {
+  render () {
+    const { t } = this.props
     return (
       <div className='auth-screen'>
         <SignInHeaderTranslated />
-        <FormSection theme={theme}>
-          <SectionBody theme={theme}>
-            <FederatedButtons
-              federated={federated}
-              theme={theme}
-              authState={authState}
-              onStateChange={onStateChange}
-              onAuthEvent={onAuthEvent}
-            />
-            <FormField theme={theme}>
-              <InputLabel>{I18n.get('Username')}</InputLabel>
-              <Input
-                data-private
-                autoFocus
-                theme={theme}
-                key='username'
-                onKeyPress={(e) => this.submitOnEnter(e)}
-                name='username'
-                onChange={this.handleInputChange}
-              />
-            </FormField>
-            <FormField theme={theme}>
-              <InputLabel>{I18n.get('Password')}</InputLabel>
-              <Input
-                data-private
-                theme={theme}
-                key='password'
-                onKeyPress={(e) => this.submitOnEnter(e)}
-                type='password'
-                name='password'
-                onChange={this.handleInputChange}
-              />
-              {
-                !hideForgotPassword && <Hint theme={theme}>
-                  {I18n.get('Forget your password? ')}
-                  <Link theme={theme} onClick={() => this.changeState('forgotPassword')}>
-                    {I18n.get('Reset password')}
-                  </Link>
-                </Hint>
-              }
-            </FormField>
-          </SectionBody>
-          <SectionFooter theme={theme}>
-            <SectionFooterPrimaryContent theme={theme}>
-              <Button theme={theme} onClick={this.signIn} disabled={this.state.loading}>
-                {I18n.get('Sign In')}
-              </Button>
-            </SectionFooterPrimaryContent>
-            {process.env.ALLOW_ANONYMOUS && <SectionFooterSecondaryContent theme={theme}>
-              {t('SignIn.AnonymousExplanation') + ' '}
-              <Link theme={theme} onClick={() => this.changeState('useAnonymous')}>
-                {t('SignIn.Anonymous')}
-              </Link>
-            </SectionFooterSecondaryContent>}
-          </SectionFooter>
-        </FormSection>
+        <div className='auth-screen__main auth-main'>
+          <div className='auth-main__anonymous-login'>
+            {t('SignIn.AnonymousExplanation') + ' '}
```

Comment:
```suggestion
            {`${t('SignIn.AnonymousExplanation') }`}
```
This doesn't really matter but I think the general pattern in this repo is to use template literals